src/components/ReadingDots.jsx

```diff
@@ -43,21 +44,16 @@ const ReadingDots = ({ actors, currentActor, currentScript }) => {
       // this helps reduce many small changes into one larger change request
       clearTimeout(scrollTimeout);
 
-      scrollTimeout = setTimeout(() => {
+      scrollTimeout = setTimeout(async () => {
```

Comment:
This doesn't need to be asynchronous, we don't need to do anything after posting the position data so we don't need to await it.